client/src/components/Dashboard.js

```diff
@@ -63,10 +67,31 @@ class Dashboard extends Component {
       });
   };
 
+  sortOnClick = (sortKey, isSorted) => {
```

Comment:
Can you make sure that only one column can have a "sort up"/"sort down" icon? Like if you sort by document first, and then by due date, the sort icon of document should be back to "sort". So visually, we know which column is used to sort the table :)

Reply:
Done